src/sgame/sg_admin.cpp

```diff
@@ -3787,7 +3787,7 @@ bool G_admin_listplayers( gentity_t *ent )
 	char            *registeredname;
 	char            lname[ MAX_NAME_LENGTH ];
 	char            bot, muted, denied;
-	int             authed = 1;
+	int             authed;
```

Comment:
nit: Make this a bool if you only ever use it as such.

Reply:
Technically this could be defined as `-1` as `pubkey_authenticated` may have 3 values
`-1` for no pubkey
`0` for not authenticated
`1` for authenticated

from: https://github.com/Unvanquished/Unvanquished/blob/6a9ad90a6741b517a757669e579ba0c1b3826b2a/src/sgame/sg_struct.h#L542

For the purposes of this PR I think this is the best way to use it?